examples/openid_connect_demo/build.rs

```diff
@@ -0,0 +1,32 @@
+use dotenvy::dotenv;
+use std::env;
+use std::fs::File;
+use std::io::Write;
+
+// https://dev.to/javiasilis/how-to-pass-environment-variables-to-a-rust-wasm-application-like-yew-dioxus-and-leptos-as-a-typescript-developer-ond
```

Comment:
This will work, but it is more complicated than we need for this example. You can pull in specific enviorment varaibles at compile time with the [env](https://doc.rust-lang.org/std/macro.env.html) macro. Because the macro resolves at compile time, it will be included as a string in the WASM binary